pocs/core.py

```diff
@@ -488,6 +498,8 @@ def _check_messages(self, queue_type, q):
     def _interrupt_and_park(self):
         self.logger.info('Park interrupt received')
         self._interrupted = True
+        # TODO(jamessynge): Should we close the dome here? Or extend the state machine
+        # with dome related states and actions?
```

Comment:
This will call `park` from any state that it is currently in and move to the `parking` state. I think it makes most sense to close the dome in the `parking` state itself and leave this as is.

Reply:
OK, I'll move it, but I'm concerned that POCS.power_down() assumes that the only device that needs to be dealt with for parking is the mount, and so makes decisions about whether to change state based on whether the mount is connected. @wtgee What do you suggest I do about that?